admin-cli/executor/client.go

```diff
@@ -65,5 +70,24 @@ func NewClient(writer io.Writer, metaAddrs []string) *Client {
 		Meta:   meta,
 		Nodes:  util.NewPegasusNodeManager(metaAddrs, replicaAddrs),
 		Perf:   aggregate.NewPerfClient(metaAddrs),
+	}, nil
+}
+
+func CloseClient(client *Client) error {
+	var errorStrings []string
+	err := client.Meta.Close()
+	if err != nil {
+		fmt.Printf("Error: failed to close meta session [%s]\n", err)
+		errorStrings = append(errorStrings, err.Error())
+	}
+
+	client.Perf.Close()
+
+	err = client.Nodes.CloseAllNodes()
+	if err != nil {
+		fmt.Printf("Error: failed to close nodes session [%s]\n", err)
+		errorStrings = append(errorStrings, err.Error())
 	}
+
+	return fmt.Errorf("%s", strings.Join(errorStrings, "\n"))
```

Comment:
Just return `nil` if there is not any error.

Reply:
Ok.